src/views/preview/preview.scss

```diff
@@ -1,14 +1,10 @@
 @import "../../colors";
 @import "../../frameless";
 
-/* stage size constants 
- * this is a hack right now - stage includes padding of .5rem (8px) for alignment in gui
- * in www the player is placed with margin -.5rem to align the edge.
- * the height is calculated from the actual height on the page (404)
-*/
-$gui-width: 496px;
+/* stage size constants */
+$gui-width: 482px;
+$gui-height: 406px;
```

Comment:
bottom of stage player seemed slightly off at 404px

Reply:
I hear you.

"gui-width" seemed to refer to the whole width of the area around the stage, while "stage-width" seemed to refer to the raw width of just the actual stage. The approx. 404px (406 here) included a bunch of other stuff besides the stage, so it seemed to fit more with "gui-" than with "stage-".

How would you feel about "player-width" and "player-height" instead of "gui-"?

Reply:
I hear you.

gui-width seems to refer to the whole area around the player; stage-width seems to refer just to the stage. 404/406 is the whole height around the player, so it seems to fit more with gui- than with stage- .

I like your "player-" idea, and used that -- what do you think?